src/main/java/com/internet/shop/dao/OrderDao.java

```diff
@@ -0,0 +1,17 @@
+package com.internet.shop.dao;
+
+import com.internet.shop.model.Order;
+import java.util.List;
+import java.util.Optional;
+
+public interface OrderDao {
+    Order create(Order order);
+
+    Optional<Order> getOrderById(Long id);
+
+    List<Order> getAll();
+
+    List<Order> getOrderByUserId(Long userId);
```

Comment:
```suggestion
    List<Order> getUserOrders(Long userId);
```
use plural in naming, you returns several orders, not one

Reply:
Done